docs/reference/metrics.rst

```diff
@@ -0,0 +1,53 @@
+.. _reference-metrics:
+
+Metrics
+=======
+
+This page lists all supported statistics that stores gather about snaps.
+
+All metrics are generated daily. Those with seven-day windows provide the average
+averages.
```

Comment:
I think this is a typo:
```suggestion
All metrics are generated daily. Those with seven-day windows provide the averages.
```

Reply:
Whoops: [98d19bc](https://github.com/canonical/snapcraft/pull/5372/commits/98d19bc604c09abf310ee8e6ea100b917829f04d)